packer/sagebio/rStudioAmi-LATEST/template.json

```diff
@@ -0,0 +1,67 @@
+{
+    "builders": [
+      {
+        "ami_name": "{{user `ImageName`}}",
+        "ami_users": "{{user `AmiUsers`}}",
+        "force_delete_snapshot": "true",
+        "instance_type": "{{user `InstanceType`}}",
+        "launch_block_device_mappings": [
+          {
+            "delete_on_termination": true,
+            "device_name": "/dev/xvda",
+            "volume_size": "{{user `VolumeSize`}}",
+            "volume_type": "gp2"
+          }
+        ],
+        "profile": "{{user `AwsProfile`}}",
+        "region": "{{user `AwsRegion`}}",
+        "snapshot_users": "{{user `SnapshotUsers`}}",
+        "source_ami": "{{user `SourceImage`}}",
+        "ssh_username": "{{user `SshUsername`}}",
+        "tags": {
+          "BuiltBy": "Packer",
+          "Department": "{{user `Department`}}",
+          "Name": "{{user `ImageName`}}",
+          "OwnerEmail": "{{user `OwnerEmail`}}",
+          "Project": "{{user `Project`}}",
+          "SourceAmiId": "{{ .SourceAMI }}",
+          "SourceAmiName": "{{ .SourceAMIName }}"
+        },
+        "type": "amazon-ebs"
+      }
+    ],
+    "post-processors": [
+      {
+        "output": "manifest.json",
+        "strip_path": true,
+        "type": "manifest"
+      }
+    ],
+    "provisioners": [
+      {
+        "ansible_env_vars": [
+          "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'",
+          "JcConnectKey={{user `JcConnectKey`}}"
+        ],
+        "playbook_file": "playbook.yaml",
+        "type": "ansible"
+      },
+      {
+        "script": "bootstrap.sh",
```

Comment:
your missing this file?  but actually all we want to do is set ENV var  `DEBIAN_FRONTEND=noninteractive` 

I think it might be better to define in the playbook in the `install packages` task.  https://stackoverflow.com/questions/27733511/how-to-set-linux-environment-variables-with-ansible